Cargo.toml

```diff
@@ -41,6 +41,7 @@ log = "0.4.14"
 maybe-async = "0.2.6"
 serde = { version = "1.0.130", default-features = false }
 serde_json = "1.0.67"
+strum = { version = "0.24.0", features = ["derive"] }
 sha2 = "0.10.0"
```

Comment:
Is it necessary to impose the `strum` crate? We are cautious about including a new crate, it will increase compile time.

Reply:
Will it really increase compile time ? As strum is built while building models which are built before the main crate. Anyway I think moving SearchFilter to the models is a good idea and will solve this problem